plugins/kotlin/idea/resources/META-INF/inspections.xml

```diff
@@ -885,6 +885,16 @@
       <category>Kotlin</category>
     </intentionAction>
 
+    <intentionAction>
+      <className>org.jetbrains.kotlin.idea.intentions.branchedTransformations.intentions.FoldIfToFunctionCallIntention</className>
+      <category>Kotlin</category>
+    </intentionAction>
+
+    <intentionAction>
+      <className>org.jetbrains.kotlin.idea.intentions.branchedTransformations.intentions.UnfoldFunctionCallToIfIntention</className>
+      <category>Kotlin</category>
```

Comment:
pls pay attention to i18n of category

it has to be (in case of `Kotlin`)
```
      <bundleName>messages.KotlinBundle</bundleName>
      <categoryKey>group.names.kotlin</categoryKey>
```

Reply:
https://github.com/JetBrains/intellij-community/pull/1673